mindjunkies/dashboard/forms.py

```
@@ -0,0 +1,9 @@
+from django import forms
+from .models import TeacherVerification, Certificate
+
+class TeacherVerificationForm(forms.ModelForm):
+    
+    class Meta:
+        model = TeacherVerification
+        fields = ['full_name', 'email', 'phone', 'address', 'portfolio_links', 
```

Ruff (0.8.2)

Trailing whitespace

Remove trailing whitespace

(W291)